lib/src/nakama_client/nakama_grpc_client.dart

```diff
@@ -299,6 +299,30 @@ class NakamaGrpcClient extends NakamaBaseClient {
     );
   }
 
+  @override
+  Future<void> updateAccount({
+    required model.Session session,
+    String? username,
+    String? displayName,
+    String? avatarUrl,
+    String? langTag,
+    String? location,
+    String? timezone,
+  }) async {
+    await _client.updateAccount(
+      UpdateAccountRequest(
+        username: username == null ? null : StringValue(value: username),
+        displayName:
+            displayName == null ? null : StringValue(value: displayName),
+        avatarUrl: avatarUrl == null ? null : StringValue(value: avatarUrl),
+        langTag: langTag == null ? null : StringValue(value: langTag),
+        location: location == null ? null : StringValue(value: location),
+        timezone: timezone == null ? null : StringValue(value: timezone),
+      ),
```

Comment:
@obrunsmann Sorry if the questions are many 😄 
I don't know what is your approach here, in protobuf it expects `StringValue` but I didn't see any usage of it, should I remove compiled code to switch from StringValue to String or keep it this way?

Reply:
No worries, just fire every question you have :) Again sorry for my delay here.

> should I edit compiled code
So **never** edit compiled code, I haven't checked this honestly speaking but I think there is no need to care about this...

Reply:
Perfect, thanks.